scripts/spack/packages/axom/package.py

```diff
@@ -7,13 +7,12 @@
 import socket
 from os.path import join as pjoin
 
-from spack import *
+from spack.package import *
```

Comment:
adding `.package` is needed now?

Reply:
I'm pretty sure that is the new spack standard. I just copy what is in the newer packages.